variod.c

```diff
@@ -361,7 +369,7 @@ int main(int argc, char *argv[])
 		fprintf(fp_console, "Connection dropped\n");	
```

Comment:
Do you see "Connection dropped" in the log file? I don't.

Reply:
It prints connection dropped each time the loop restarts.
It also closed the socket and reopened it with each iteration.
What gave me a clue of the socket problem was Xcsoar log.
End of file error in port 4352. This indicated that variod was killing the port before XCSoar was able to finish data reception.
After this changes this error was fixed and command communication started working reliably again.